src/main/java/List.java

```diff
@@ -0,0 +1,167 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
+public class List{
+    //variable
+    private ArrayList<String> list;
+    private ArrayList<Boolean> done;
+    private ArrayList<Character> task_cat;
+    private static int item_count = 0;
```

Comment:
Rather than this, use a more OOP approach where you keep a list of Task objects.